### PR TITLE
Add multiple numbers support, update README and add requirements.txt

### PR DESCRIPTION
**Pull request by aditeyabaral**
Hi, I've added support for multiple numbers in the URL.
The usage remains the same: `/value-1/value-2/value-n`

### CODE REVIEW

**Comment by anirudhRowjee** [comment]
Perhaps optimizing the backend API at `add.woohoojin.dev` might be a better idea, but this implementation beats some of the best ways to do this that I've seen.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Flask-Cors==3.0.9
+Flask-Limiter==1.4
+itsdangerous==1.1.0
+Jinja2==2.11.2
+limits==1.5.1
+MarkupSafe==1.1.1
+pycodestyle==2.6.0
+six==1.15.0
+toml==0.10.2
+Werkzeug==1.0.1
```
Comment (crscillitoe) [inline requirements.txt]: There's a lot of packages in this requirements.txt that don't appear relevant.

**Review by crscillitoe — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (aditeyabaral) [reply requirements.txt]: I have removed the dependency libraries.

**Comment by aditeyabaral** [comment]
I have made the requested changes

**Comment by crscillitoe** [comment]
Just seems useless. Why would you want to add 3 numbers together when you can just add 2 numbers together.

**Pull request closed without merging**